content/docs/command-reference/metrics/index.md

```diff
@@ -69,10 +69,10 @@ stages:
 
 ### Supported file formats
 
-Metrics can be organized as tree hierarchies in JSON or YAML files. DVC
-addresses specific metrics by the tree path. In the JSON example below, five
-metrics are presented: `train.accuracy`, `train.loss`, `train.TN`, `train.FP`
-and `time_real`.
+Metrics can be organized as tree hierarchies in JSON or YAML files (specifically
+in YAML 1.2 format). DVC addresses specific metrics by the tree path. In the
+JSON example below, five metrics are presented: `train.accuracy`, `train.loss`,
+`train.TN`, `train.FP` and `time_real`.
```

Comment:
```suggestion
Metrics can be organized as tree hierarchies in JSON or YAML 1.2 files. DVC
addresses specific metrics by the tree path. In the JSON example below, five
metrics are presented: `train.accuracy`, `train.loss`, `train.TN`, `train.FP`
and `time_real`.
```

Reply:
Linking "YAML 1.2" to https://yaml.org/spec/1.2/spec.html

Reply:
Actually (per https://github.com/iterative/dvc.org/issues/1704#issuecomment-677425269) probably just "YAML" (but linked) is enough.

> Same in the other instances of this suggestion.

Reply:
Let's not link it, it just adds noise to the docs.

Reply:
"YAML 1.2 files" might be confusing to the users as it might send them to a different rabbit hole than just writing yaml files. Sprinkling these on key areas as a small note might be a good idea.

Reply:
> "YAML 1.2 files" might be confusing to the users as it might send them to a different rabbit

How is "YAML files (specifically in YAML 1.2 format)" different? It just uses 3x words 🙂